src/transformers/models/prophetnet/modeling_prophetnet.py

```diff
@@ -749,14 +749,14 @@ def forward(self, hidden_states):
         return hidden_states
 
 
-class ProphetNetNgramProphetNetSelfAttention(nn.Module):
+class ProphetNetNgramSelfAttention(nn.Module):
```

Comment:
I think it's ok to do the renaming here without deprecation warning since it's not a public class.